### PR TITLE
Update download.md

### PR DESCRIPTION
**Pull request by simonschaufi**
* change download links to zip archive
* remove redirect to english demo website

### CODE REVIEW

File: download.md [hunk 1]
```diff
@@ -11,9 +11,9 @@ redirect_from:
 # Downloads
 
 <section id="downloads">
-      <a href="{{ site.stable_url }}"><span>Stable<br/>0.9.3</span></a>
-      <a href="https://github.com/kimai/kimai/zipball/master"><span>DEV<br/>0.9.4</span></a>
-      <a href="https://github.com/kimai/manuals/raw/master/documentation.pdf"><span>Docu<br/>PDF</span></a>
+  <a href="{{ site.stable_url }}"><span>Stable<br/>0.9.3</span></a>
+  <a href="https://github.com/kimai/kimai/archive/master.zip"><span>DEV<br/>0.9.4</span></a>
```
Comment (kevinpapst) [inline download.md]: Is there a difference beside the filename? Both links spit out the version with the latest changes in INSTALL.md

File: download.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ When you update an existing installation, just replace the entire Kimai folder o
 
 For current versions (&gt;0.8.0) when you installed Kimai via the UI-Installer (not by  editing the ‘conf.php’) you need to keep the file  ‘autoconf.php’. For older versions (at least version 0.7) keep the file ‘includes/conf.php’.
 
-When Kimai doesn’t start the reason is probably wrong writing permissions on the mentioned files and folders! Users also encountered problems with older PDO version – see support forum for more detail
+When Kimai doesn’t start the reason is probably wrong writing permissions on the mentioned files and folders! Users also encountered problems with older PDO version – see support forum for more detail.
```
Comment (kevinpapst) [inline download.md]: PDO is not supported any longer, could be deleted

File: download.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ When you update an existing installation, just replace the entire Kimai folder o
 
 For current versions (&gt;0.8.0) when you installed Kimai via the UI-Installer (not by  editing the ‘conf.php’) you need to keep the file  ‘autoconf.php’. For older versions (at least version 0.7) keep the file ‘includes/conf.php’.
```
Comment (kevinpapst) [inline download.md]: That could be deleted completely, way to old to still support

**Comment by kevinpapst** [comment]
Can't find the change "remove redirect to english demo website" - did you forget to commit it?

File: download.md (hunk 1 above)
Comment (simonschaufi) [inline download.md]: just a more beautiful file name ;)

**Comment by simonschaufi** [comment]
look at line 32.

**Comment by kevinpapst** [comment]
Ah, there it is ;-)

Thanks for the updates

**Pull request merged**